apps/src/javalab/Javalab.js

```diff
@@ -94,6 +96,7 @@ Javalab.prototype.init = function (config) {
 
   this.skin = config.skin;
   this.level = config.level;
+  this.levelIdForAnalytics = config.serverLevelId;
```

Comment:
I'm unclear about why we have `config.level.id` and `config.levelId` and `config.serverLevelId`, but `level.id` and `levelId` are both set to "custom" and only the `serverLevelId` is the actual number associated with the level.